notebooks/BikeShareExample.ipynb

```diff
@@ -574,33 +576,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "deployedPredictorsResponse=forecast.list_deployed_predictors()\n",
-    "print(deployedPredictorsResponse)"
+    "create_forecast_response=forecast.create_forecast(ForecastName=forecastName,\n",
+    "                                                  PredictorArn=predictorArn)\n",
+    "forecastArn = create_forecast_response['ForecastArn']"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Please note that the following cell can also take 10 minutes or more to be fully operational. There's no output here, but that is fine as long as the * is there."
+    "Check the status of the forecast process, when the status change from **CREATING** to **ACTIVE**, we can continue to next steps. Depending on data size, model selection and hyper parameters，it can take 10 mins to more than one hour to be **ACTIVE**."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "while True:\n",
-    "    deployedPredictorStatus = forecast.describe_deployed_predictor(PredictorName=predictorName)['Status']\n",
-    "    print(deployedPredictorStatus)\n",
-    "    if deployedPredictorStatus != 'ACTIVE' and deployedPredictorStatus != 'FAILED':\n",
+    "    forecastStatus = forecast.describe_forecast(ForecastArn=forecastArn)['Status']\n",
+    "    print(forecastStatus)\n",
+    "    if forecastStatus != 'ACTIVE' and forecastStatus != 'FAILED' and forecastStatus != 'CREATE_FAILED':\n",
```

Comment:
forecastStatus != FAILED can be removed.

Reply:
Done